app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -21,9 +21,16 @@ class MeetingRequestViewModel(
 ) : ViewModel() {
 
   var meetingRequestState by mutableStateOf(MeetingRequest())
+  var meetingResponseState by mutableStateOf(MeetingResponse())
+  var meetingConfirmationState by mutableStateOf(MeetingConfirmation())
 
-  private val SEND_MESSAGE_FUNCTION_NAME = "sendMessage"
   private val SEND_MEETING_REQUEST = "sendMeetingRequest"
+  private val SEND_MEETING_RESPONSE = "sendMeetingResponse"
+  private val SEND_MEETING_CONFIRMATION = "sendMeetingConfirmation"
```

Comment:
these are constants.

Reply:
True, I ll put them at the begging of the file